### PR TITLE
Legend Creation

### PR DESCRIPTION
**Pull request by pranathibora14**
Created a legend similar to the resources repo mentioned in the issue. Let me know if any changes are required.

### CODE REVIEW

**Review by DiptoChakrabarty — COMMENTED** [review]
cli commands , short hands etc are also required in the index

**Review by akshatvg — CHANGES_REQUESTED** [review]
I see you had added the GitHub CLI and other stuff to the legend but then removed it? Could you re-add it?

The rest looks good.

File: README.md [hunk 1]
```diff
@@ -2,6 +2,16 @@
...
 
+## Index :books:
+* [Basic Commands :v:](#basic-commands-v)
+* [Branches :deciduous_tree:](#branches-deciduous_tree)
+* [Update forked Repository with original Repository :hourglass:](#update-forked-repository-with-original-repository--hourglass)
+* [Dev branch is X commits behind and Y commits ahead of master](#master)
+* [Short hands](#shorthands)
+* [GitHub CLI commands](#cli)
+
+
```
Comment (DiptoChakrabarty) [inline README.md]: It seems that these links are broken please fix them

Comment (pranathibora14) [reply README.md]: Okay sure! I'll add them back in.

**Review by DiptoChakrabarty — APPROVED** [review]

**Comment by DiptoChakrabarty** [comment]
Looks good

**Pull request merged**